File: EjerciciosEPI/3_operacion/operacion.c
#include <stdio.h>
#include <math.h>

int main () {
	int limite;
	double suma;
	double media;

	suma = 0;
	media = 0;

	printf("Introduzca el limite del sumatorio: ");
	scanf("%d", &limite);

	for (int i = 0 ; i <= 100 ; i++)
	{
		media+=i;
	}
	media/=100;

	printf("Media: %f", media);

	for (int i = 1 ; i <= limite ; i++)
	{
		suma+=(pow(media, i+1)/sqrt(i));
	}

	printf("El sumatorio es: %f", suma);
	return 0;
}

File: Ejercicios/EjerciciosEPI/3_operacion/operacion.py
"""
Programa que haga el siguiente sumatorio: (n^(i+1)/sqrt(i)), siendo n la media de los numeros desde el 0 hasta el 100 -ambos incluidos-.
El sumatorio comienza en 1 y finaliza en un numero introducido por el usuario.
Ejemplo:
    Introduzca el limite del sumatorio: 20
    El sumatorio es: 1.34...
"""

import math

suma = 0
limite = 0
media = 0

limite = int(input("Introduzca el limite del sumatorio: "))

for i in range (0, 101, 1):
    media = media + i

media = media / 100
print(media)
    
for i in range (1, limite+1, 1):
    suma = suma + (media**(i+1)/math.sqrt(i))

print("El sumatorio es:", suma)
